### ISSUE TITLE
Normalization of objective function value 

### ISSUE THREAD

**Issue by mahdizibrahim** [optuna/optuna#5283]
### Motivation

I am doing Multi-objective hyperparameter optimization of artificial neural network using Pytorch. 
After solving the optimization problem using NSGA-III sampler in Optuna, I want to visualize the pareto front plot.
However, the 3D objective functions values of the solved trails are spread (scattered), so the plot is displaying 2 points (points on the left and points on the right)

### Description

The solution is to normalize the values of the objective function in the study that has already been done and saved.(using Z-score for example)
This will allow the programmer to normalize the values of the objective function if needed and to visualize the pareto front even if the distance between the solutions is very distinct.

**Comment by nzw0301** [comment]
For visualization, we can change objective values by specifying the `targets` argument. In this case, we specify a function to normalise objective values somehow.

**Comment by nabenabe0928** [comment]
Thank @nzw0301 , Here is an example for @mahdibrahim :

```python
import optuna
import numpy as np


def objective(trial):
    x1 = trial.suggest_float("x1", -5, 5)
    x2 = trial.suggest_float("x2", -5, 5)
    return x1 ** 2 + x2 ** 2, (x1 - 2) ** 2 + (x2 + 3) ** 2, (x1 - 3) ** 2 + (x2 - 2) ** 2


study = optuna.create_study(directions=["minimize"] * 3)
study.optimize(objective, n_trials=30)

values = np.array([t.values for t in study.trials])
v_max = np.max(values, axis=0)
v_min = np.min(values, axis=0)
fig = optuna.visualization.plot_pareto_front(
    study,
    targets=lambda trial: list((np.array(trial.values) - v_min) / (v_max - v_min)),
)
fig.show()

```
